packages/e/erc725js.json

```
@@ -0,0 +1,55 @@
+{
+  "name": "erc725.js",
+  "filename": "erc725.min.js",
+  "description": "Library to interact with ERC725 smart contracts and encode/decode ERC725Y data keys.",
+  "homepage": "https://docs.lukso.tech/tools/erc725js/getting-started",
+  "keywords": [
+    "ethereum", 
+    "erc725", 
+    "lsp",
+    "lukso"
+  ],
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ERC725Alliance/erc725.js/tree/develop"
+  },
+  "autoupdate": {
+    "source": "npm",
+    "target": "@erc725/erc725.js",
+    "fileMap": [
+      {
+        "basePath": "",
```

```suggestion
        "basePath": "build/main/src",
```
The files under `build/main/src` make use of Node.js `require` calls and will not work in a browser. As such, I suspect you just want the files from `build/module/src`, which should work in browsers that support ESM.